static/src/stylesheets/module/_adslot.scss

```diff
@@ -188,6 +188,7 @@
  */
 .ad-slot--inline,
 .ad-slot--container-inline,
+.ad-slot--container-inline.ad-slot--fluid,
```

Comment:
This file needs some serious rethinking, more generally the naming strategy for ad slots, creatives and the surrounding context.